packages/web/src/pages/profile-page/components/desktop/ProfileTopTags.tsx

```diff
@@ -20,48 +13,28 @@ const messages = {
 const MOST_USED_TAGS_COUNT = 5
 
 export const ProfileTopTags = () => {
-  const dispatch = useDispatch()
   const user = useProfileParams()
-  const handle = user?.handle
-  const userId = user?.user_id
-
-  const topTagsStatus = useSelector((state: AppState) => {
-    if (!handle) return Status.IDLE
-    return getTopTagsStatus(state, handle)
-  })
+  const { data, status } = useTopTags(user?.user_id)
+  const topTags = data?.slice(0, MOST_USED_TAGS_COUNT)
```

Comment:
does the topTags endpoint take a limit? if so, we could just pass that into the hook

Reply:
i dont believe so bc the saga did this too, but i'll check

Reply:
it does 🫢